src/pudl/convert/ferc_to_sqlite.py

```diff
@@ -108,52 +108,33 @@ def main():  # noqa: C901
         pudl_in=pudl_in, pudl_out=pudl_out
     )
 
-    # Use sandbox arg from CLI
+    parsed_settings = FercToSqliteSettings().parse_obj(
+        script_settings["ferc_to_sqlite_settings"]
+    )
+
+    # Configure how we want to obtain raw input data:
+    ds_kwargs = dict(
+        gcs_cache_path=args.gcs_cache_path, sandbox=pudl_settings.get("sandbox", False)
+    )
+    if not args.bypass_local_cache:
+        ds_kwargs["local_cache_path"] = Path(pudl_settings["pudl_in"]) / "data"
+
     pudl_settings["sandbox"] = args.sandbox
+    pudl.extract.ferc1.dbf2sqlite(
+        ferc1_to_sqlite_settings=parsed_settings.ferc1_dbf_to_sqlite_settings,
+        pudl_settings=pudl_settings,
+        clobber=args.clobber,
+        datastore=Datastore(**ds_kwargs),
+    )
 
-    # Perform DBF to SQLite conversion if requested
-    if "ferc1_dbf_to_sqlite_settings" in script_settings:
-        dbf_settings = Ferc1DbfToSqliteSettings().parse_obj(
-            script_settings["ferc1_dbf_to_sqlite_settings"]
-        )
-
-        # Configure how we want to obtain raw input data:
-        ds_kwargs = dict(
-            gcs_cache_path=args.gcs_cache_path,
-            sandbox=pudl_settings.get("sandbox", False),
-        )
-        if not args.bypass_local_cache:
-            ds_kwargs["local_cache_path"] = Path(pudl_settings["pudl_in"]) / "data"
-
-        pudl.extract.ferc1.dbf2sqlite(
-            ferc1_to_sqlite_settings=dbf_settings,
-            pudl_settings=pudl_settings,
-            clobber=args.clobber,
-            datastore=Datastore(**ds_kwargs),
-        )
-    else:
-        # Notify user that no DBF data was requested
-        logger.info("No FERC 1 DBF to SQLite settings found.")
-
-    # Perform XBRL to SQLite conversion if requested
-    if "ferc1_xbrl_to_sqlite_settings" in script_settings:
-        xbrl_settings = Ferc1XbrlToSqliteSettings().parse_obj(
-            script_settings["ferc1_xbrl_to_sqlite_settings"]
-        )
-
-        pudl.extract.ferc1.xbrl2sqlite(
-            ferc1_to_sqlite_settings=xbrl_settings,
-            pudl_settings=pudl_settings,
-            clobber=args.clobber,
-            datastore=Datastore(
-                local_cache_path=(Path(pudl_in) / "data"), sandbox=True
-            ),
-            batch_size=args.batch_size,
-            workers=args.workers,
-        )
-    else:
-        # Notify user that no XBRL data was requested
-        logger.info("No FERC 1 XBRL to SQLite settings found.")
+    pudl.extract.xbrl.xbrl2sqlite(
+        ferc_to_sqlite_settings=parsed_settings,
+        pudl_settings=pudl_settings,
+        clobber=args.clobber,
+        datastore=Datastore(local_cache_path=(Path(pudl_in) / "data"), sandbox=True),
```

Comment:
The Datastore used `dbf2sqlite` is pulling its kwargs from the `ds_kwargs` dictionary, but here the kwargs are hard coded, so they could be different, leading to `dbf2sqlite` and `xbrl2sqlite` getting their raw inputs from different places. That seems confusing. Is there a reason they can't use the same kwargs?

If you need to ensure that `sandbox=True` until the Datastore is fixed to allow either sandbox or production DOIs, then maybe just that one element could be overridden for the arguments here.

Reply:
Fixed to use same `ds_kwargs`, and explicitly set `sandbox` to true. The `Datastore` updates are still on my radar, but felt like too much for this PR